### ISSUE TITLE
[JOSS] Paper review

### ISSUE THREAD

**Issue by matbesancon** [Non-Contradiction/JuliaCall#95]
Last minor points on the paper itself:

```
There are R packages which wrap (no S)
for Disciplined Convex Programming ... providing (no by)
which utilizes DifferentialEquations.jl ->  using DifferentialEquations.jl
and the Julia package RCall
The package RJulia (Gong, Haverty...
```

**Comment by Non-Contradiction** [comment]
Thank you very much! The problems are fixed by the commit f09295a.

**Comment by matbesancon** [comment]
Thanks perfect
